portfolio/src/main/webapp/script.js

```diff
@@ -13,17 +13,45 @@
 // limitations under the License.
 
 
+function getComments() {
+
+  document.getElementById('greeting-container').innerHTML = "";
+  var commentNum = parseInt(document.getElementById('quantity').value);
+  
+
```

Comment:
nit: You can remove one of these blank lines.

Reply:
and Done!